domain/src/main/java/org/oppia/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -1,96 +1,244 @@
-      FRACTIONS_EXPLORATION_ID_1
+      FRACTIONS_EXPLORATION_ID_1,
+      timestamp
     )
   }
 
   /** Marks one story progress full in ratios exploration for a particular profile. */
-  fun markFullStoryPartialTopicProgressForRatios(profileId: ProfileId) {
+  fun markFullStoryPartialTopicProgressForRatios(profileId: ProfileId, timestampOlderThanAWeek: Boolean) {
+    val timestamp = if (!timestampOlderThanAWeek) {
+      getCurrentTimestamp()
+    } else {
+      getOldTimestamp()
+    }
     storyProgressController.recordCompletedChapter(
       profileId,
       RATIOS_TOPIC_ID,
       RATIOS_STORY_ID_0,
-      RATIOS_EXPLORATION_ID_0
+      RATIOS_EXPLORATION_ID_0,
+      timestamp
     )
 
     storyProgressController.recordCompletedChapter(
       profileId,
       RATIOS_TOPIC_ID,
       RATIOS_STORY_ID_0,
-      RATIOS_EXPLORATION_ID_1
+      RATIOS_EXPLORATION_ID_1,
+      timestamp
     )
   }
 
   /** Marks two partial story progress in ratios exploration for a particular profile. */
-  fun markTwoPartialStoryProgressForRatios(profileId: ProfileId) {
+  fun markTwoPartialStoryProgressForRatios(profileId: ProfileId, timestampOlderThanAWeek: Boolean) {
+    val timestamp = if (!timestampOlderThanAWeek) {
+      getCurrentTimestamp()
+    } else {
+      getOldTimestamp()
+    }
     storyProgressController.recordCompletedChapter(
       profileId,
       RATIOS_TOPIC_ID,
       RATIOS_STORY_ID_0,
-      RATIOS_EXPLORATION_ID_0
+      RATIOS_EXPLORATION_ID_0,
+      timestamp
     )
 
     storyProgressController.recordCompletedChapter(
       profileId,
       RATIOS_TOPIC_ID,
       RATIOS_STORY_ID_1,
-      RATIOS_EXPLORATION_ID_2
+      RATIOS_EXPLORATION_ID_2,
+      timestamp
+    )
+  }
+
+  /** Marks exploration [FRACTIONS_EXPLORATION_ID_0] as recently played for a particular profile. */
+  fun markRecentlyPlayedForFractionsStory0Exploration0(profileId: ProfileId, timestampOlderThanAWeek: Boolean) {
+    val timestamp = if (!timestampOlderThanAWeek) {
+      getCurrentTimestamp()
+    } else {
+      getOldTimestamp()
+    }
+    storyProgressController.recordRecentlyPlayedChapter(
+      profileId,
+      FRACTIONS_TOPIC_ID,
+      FRACTIONS_STORY_ID_0,
+      FRACTIONS_EXPLORATION_ID_0,
+      timestamp
+    )
+  }
+
+  /** Marks exploration [RATIOS_EXPLORATION_ID_0] as recently played for a particular profile. */
+  fun markRecentlyPlayedForRatiosStory0Exploration0(profileId: ProfileId, timestampOlderThanAWeek: Boolean) {
+    val timestamp = if (!timestampOlderThanAWeek) {
+      getCurrentTimestamp()
+    } else {
+      getOldTimestamp()
+    }
+    storyProgressController.recordRecentlyPlayedChapter(
+      profileId,
+      RATIOS_TOPIC_ID,
+      RATIOS_STORY_ID_0,
+      RATIOS_EXPLORATION_ID_0,
+      timestamp
+    )
+  }
+
+  /** Marks first exploration in both stories of Ratios as recently played for a particular profile. */
+  fun markRecentlyPlayedForRatiosStory0Exploration0AndStory1Exploration2(
```

Comment:
Here you marking for 1st exploration but this method contains Exploration0 and Exploration2. This method is not clearly stating its context. Please check if I am confused correct me.

Reply:
RATIOS_TOPIC_ID - Ratios
--- RATIOS_STORY_ID_0
-------- RATIOS_EXPLORATION_ID_0
-------- RATIOS_EXPLORATION_ID_1
--- RATIOS_STORY_ID_1
-------- RATIOS_EXPLORATION_ID_2
-------- RATIOS_EXPLORATION_ID_3

So basically we are marking first exploration of both the stories as marked and the IDs corresponding to that are `RATIOS_EXPLORATION_ID_0` and `RATIOS_EXPLORATION_ID_2`

Reply:
Ah okay, got it.